README.md

```diff
@@ -1 +1,10 @@
 # TDR Backend Checks Utils
+
+The TDR backend checks are now running in a step function. 
+There is a size limit of 256Kb for messages passed between steps and for large consignments, the message is more than this.
+
+To solve this, each step reads its input from S3 and once it has carried out its step, writes the new output to S3 again.
```

Comment:
Could you mention that AWS recommends this approach of using s3 for large step messages, and reference the URL where this is stated.